src/OpenTelemetry.Instrumentation.StackExchangeRedis/OpenTelemetryBuilderExtensions.cs

```diff
@@ -0,0 +1,53 @@
+﻿// <copyright file="OpenTelemetryBuilderExtensions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using OpenTelemetry.Instrumentation.StackExchangeRedis;
+using StackExchange.Redis;
+
+namespace OpenTelemetry.Trace.Configuration
+{
+    /// <summary>
+    /// Extension methods to simplify registering of dependency instrumentation.
+    /// </summary>
+    public static class OpenTelemetryBuilderExtensions
+    {
+        /// <summary>
+        /// Enables the outgoing requests automatic data collection for all supported activity sources.
```

Comment:
incorrect comment for Redis

Reply:
updated to: `Enables the outgoing requests automatic data collection for Redis`